app/uk/gov/hmrc/crdlcacheadminfrontend/config/AppConfig.scala

```diff
@@ -1,10 +1,32 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package uk.gov.hmrc.crdlcacheadminfrontend.config
 
 import javax.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()(config: Configuration):
-
-  val welshLanguageSupportEnabled: Boolean =
+class AppConfig @Inject(servicesConfig: ServicesConfig)(val config: Configuration) {
+  lazy val welshLanguageSupportEnabled: Boolean =
     config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
+
+  lazy val crdlCacheUrl: String = s"${servicesConfig.baseUrl("crdl-cache")}/crdl-cache"
+  lazy val emcsTFERefDataUrl: String = s"${servicesConfig.baseUrl("emcs-tfe-crld-ref-data")}"
```

Comment:
```suggestion
  lazy val emcsTFERefDataUrl: String = s"${servicesConfig.baseUrl("emcs-tfe-crdl-reference-data")}"
```
I know it's a bit annoying but we usually use the full service name in `servicesConfig` (plus it allows the MDTP Catalogue to link the services together).